index.html

```diff
@@ -0,0 +1,170 @@
+            <i class="fa-sharp fa-solid fa-book-open-reader"></i>
+            <h3>Lecture</h3>
+            <p>Listen to the speakers from various countries
+              about the messages of sharing and opening.</p>
+          </div>
+        </article>
+        <article class="bg-color list-prog1">
+          <div class="insidemain flex-center">
+            <i class="fa-brands fa-accusoft"></i>
+            <h3>CC Exhibition</h3>
+            <p>Appreciate various creations applying
+              CC license of artists, organized from Art Center</p>
+          </div>
+        </article>
+        <article class="bg-color list-prog1">
+          <div class="insidemain flex-center">
+            <i class="fa-regular fa-comments"></i>
+            <h3>Forum</h3>
+            <p>Have the time to share your thoughts
+              and opinions with experts for each topic.</p>
+          </div>
+        </article>
+        <article class="bg-color list-prog1">
+          <div class="insidemain flex-center">
+            <i class="fa-solid fa-code-branch"></i>
+            <h3>Workshop</h3>
+            <p>Try creating your own work using open
+              source license rather than just watching at it.</p>
+          </div>
+        </article>
+        <article class="bg-color list-prog1">
+          <div class="insidemain flex-center">
+            <i class="fa-solid fa-gears"></i>
+            <h3>CC Ignite</h3>
+            <p>get opportunities to network with CC
+              affiliates around the world, also after the summit.</p>
+          </div>
+        </article>
+      </div>
+      <button type="button" name="join" class="btn-join">Join the Program</button>
+      <a class="see-pro"><u>THE WHOLE PROGRAM</u></a>
+      </div>
+    </section>
+
+    <section class="speakers flex-center" id="speakers">
+      <h2>Featured Speakers</h2>
+      <hr class="line">
+      <div class="speakers1" id="contspeak">
+        <!-- This item now in java-script -->
+      </div>
+      <button type="button" name="join" class="btn">MORE <i class="fas fa-angle-down down-arrow"></i></button>
+    </section>
+
+    <section class="partner flex-center" id="partner">
+      <h2>Partner</h2>
+      <hr class="line">
+      <div class="partner-link flex-center">
+        <a href="#"><img src="assests/sponcers/Pfizer.svg.png" alt="pfz-img"></a>
+        <a href="#"><img src="assests/sponcers/disney.png" alt="disney-img"></a>
+        <a href="#"><img src="assests/sponcers/Rolex_logo.svg.png" alt="rolex-img"></a>
+      </div>
+    </section>
+  </main>
+  <footer class="contact flex-center">
+    <h2>Contact</h2>
+    <hr class="line">
+    <div class="foot-cont">
+      <img src="assests/logo/logo.png" alt="img">
+      <p>THE 95TH ACADEMY AWARDS | 2023
+        Dolby Theatre at Ovation Hollywood
+        Sunday, March 12, 2023
+        Honoring movies released in 2022</p>
+    </div>
+    <p class="right">&copy; 2023 All Rights Reserved</p>
+  </footer>
```

Comment:
- [ ] The request from the previous reviewer about the alignment of the footer logo and text has not been implemented. Kindly attend to that.

- [OPTIONAL] When you look at both partner and footer section, you could see that they both have the same height. Consider your desgin to look the same.

### Your Design
[your-partner-footer.webm](https://user-images.githubusercontent.com/26597432/225530518-f179220e-a796-4ce6-855b-8a996e1348d1.webm)

### Figma Design
![temp-partner-footer](https://user-images.githubusercontent.com/26597432/225530548-1fa610d5-3c28-4083-8340-1bd86be8b77d.png)

Reply:
👍